src/components/NotFound.test.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import NotFound from "./NotFound";
+
+describe("NotFound component", () => {
+  it("renders the NotFound component with the correct image", () => {
+    render(<NotFound />);
+
+    const notFoundImage = screen.getByAltText("404 Not Found");
+
+    expect(notFoundImage).toBeInTheDocument();
+  });
```

Comment:
Since NotFound is in the pages directory, the test should be in the pages directory too.